pkg/bff/gds_test.go

```diff
@@ -62,70 +65,85 @@ func (s *bffTestSuite) TestLookup() {
 	require.NotEmpty(rep.TestNet, "expected testnet result from server")
 }
 
-func (s *bffTestSuite) TestRegister() {
+func (s *bffTestSuite) TestLoadRegisterForm() {
```

Comment:
Hopefully this is a good template for doing tests against our API endpoints with the BFF client.